configure

```diff
@@ -28,6 +28,17 @@ check_sndio()
 	return 1
 }
 
+check_coreaudio()
+{
+	case `uname -s` in
+	Darwin)
```

Comment:
OpenDarwin/PureDarwin also use this name. But I doubt if there's anyone using these OSes.